content/200-concepts/200-database-connectors/08-sql-server/020-sql-server-local.mdx

```diff
@@ -14,7 +14,7 @@ To run a Microsoft SQL Server locally on a Windows machine:
 
 1. Use Windows Authentication to log in to Microsoft SQL Server Management Studio (expand the **Server Name** dropdown and click **&lt;Browse for more...&gt;** to find your database engine):
 
-![The New Query button in SQL Server Management Studio](../../../../doc-images/connect-sql-server.png)
+![The New Query button in SQL Server Management Studio](../../../doc-images/connect-sql-server.png)
```

Comment:
Uh, was that broken in the current docs version?

Reply:
Yep